### DISCUSSION TITLE
nginx_recipes docker mount points

### DISCUSSION THREAD

**Discussion by bartekd123** [TandoorRecipes/recipes#1163] [Q&A] [answered]
I am setting up tandoor, and can not wrap my head around the nginx web server part. In the documentation you write this:

This has the downside that it is difficult to supply the configuration to the webserver (e.g. nginx). Up until version 0.13.0, this had to be done manually by downloading the nginx config file and placing it in a directory that was then mounted into the nginx container.

I see all my containers starting up, and when I set a port for the web_recipes container, i can access it directly. I can also shell into the nginx_recipes and see that mediafiles and staticfiles are accessible within the container (under /media and /static. I can also cd to /etc/nginx/conf.d/ and see that it is set to present the default nginx welcome page.

Is there something else i need to do to specify the nginx config? With the above note, I can't manual add things to the config file. I feel I am missing something obvious, but don't see it in the documentation, faq or previous issues in github.

**Comment by vabene1111** [comment, marked as answer]
i dont think you are missing anything, the nginx config is stored in a volume, if you want to edit it you need to go into that volume or change the mount point. 
If you just use the default docker compose and nginx configuration this shoudl all work automatically and should also be possible to integrate this into any proxy setup you like without making changes to the config. 

Or am i missing a point ?

**Reply by bartekd123** [reply]
oh ok that may be it then. I used the default setup except the following:
 - instead of using the .env file, I declared the variables directly
 - instead of using volumes, I used direct mounts since I want all my docker containers on a different drive

Can you let me know what config is needed for nginx to be the webserver in the standard config if I go this way?

**Reply by vabene1111** [reply]
this one https://github.com/TandoorRecipes/recipes/blob/develop/nginx/conf.d/Recipes.conf

**Reply by bartekd123** [reply]
Thanks, I downloaded the file above into the nginx_config folder and mounted the folder like below:
      - /DockerLocation/recipes/nginx_config:/etc/nginx/conf.d:ro

All is good now, thanks for your help @vabene1111
